lib/Pod/To/Text.rakumod

```diff
@@ -49,9 +49,9 @@ sub item2text($pod) {
 sub named2text($pod) {
     given $pod.name {
         when 'pod'  { pod2text($pod.contents)     }
-        when 'para' { para2text($pod.contents[0]) }
+        when 'para' { pod2text($pod.contents) }
```

Comment:
OOC, is para2text still referred to in other places?

Reply:
Actually no afaik, good catch. Though given the naming conventions here: https://github.com/rakudo/rakudo/blob/178a00f86fe830c09be338a7e3b6f2f55a7f475a/lib/Pod/To/Text.rakumod#L13-L31 para2text should probably be called from line 18, with the code currently there in para2text instead. If that looks right to you, I'll add a commit for it to this PR.

Reply:
I decided to take the silence as agreement :)